lesson_01/homework_01.py

```diff
@@ -1,32 +1,34 @@
 # task 01 == Виправте синтаксичні помилки
 print("Hello", end = " ")
-    print("world!")
+print("world!")
 
 # task 02 == Виправте синтаксичні помилки
 hello = "Hello"
 world = "world"
 if True:
-print(f"{hello} {world}!")
+    print(f"{hello} {world}!")
 
 # task 03  == Вcтавте пропущену змінну у ф-цію print
+
 for letter in "Hello world!":
-    print()
+    print(letter, end="")
+
 
 # task 04 == Зробіть так, щоб кількість бананів була
 # завжди в чотири рази більша, ніж яблук
 apples = 2
-banana = x
+banana = apples * 4
 
 # task 05 == виправте назви змінних
-1_storona = 1
-?torona_2 = 2
-сторона_3 = 3
-$torona_4 = 4
+side_1 = 1
+side_2 = 2
+side_3 = 3
+side_4 = 4
 
 # task 06 == Порахуйте периметр фігури з task 05
 # та виведіть його для користувача
-perimetery = ? + ? + ? + ?
-print()
+perimeter = side_1 + side_2 + side_3 + side_4
+print(perimeter)
```

Comment:
Кстаті, не звернув увагу але тут по англійськи вірно саме з у на кінці
```suggestion
perimetery = side_1 + side_2 + side_3 + side_4
print(perimetery)
```